package.json

```diff
@@ -125,11 +126,11 @@
     "zxcvbn": "4.4.2"
   },
   "devDependencies": {
-    "@babel/core": "7.16.0",
-    "@babel/eslint-parser": "7.21.8",
-    "@babel/plugin-proposal-export-default-from": "7.16.0",
-    "@babel/preset-env": "7.16.0",
-    "@babel/preset-react": "7.16.0",
+    "@babel/core": "7.22.5",
```

Comment:
IS this change absolutely necessary? If not I'd say we drop it to narrow down the scope.

Reply:
I can test if it works without this update. It is recommended.